flaps/flaps_volumes.go

```diff
@@ -25,6 +26,21 @@ func (f *Client) GetVolumes(ctx context.Context) ([]api.Volume, error) {
 	return out, nil
 }
 
+func (f *Client) GetVolumes(ctx context.Context) ([]api.Volume, error) {
+	volumes, err := f.GetAllVolumes(ctx)
+	if err != nil {
+		return nil, err
+	}
+	return lo.Filter(volumes, func(v api.Volume, _ int) bool {
+		switch v.State {
+		case "scheduling_destroy", "fork_cleanup", "waiting_for_detach", "pending_destroy", "destroying":
```

Comment:
Could also use the slices.Contains function and make the slices of invalid states a const/var.